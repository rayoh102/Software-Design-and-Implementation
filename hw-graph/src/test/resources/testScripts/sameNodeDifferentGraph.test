# Tests if we can create multiple instances of the same graph

# Create a graph
CreateGraph graph1

# Create another graph
CreateGraph graph2

# Add a node to first graph
AddNode graph1 n1

# Add same node to second graph
AddNode graph2 n1

# Repeat for another node
AddNode graph1 n2
AddNode graph2 n2

# Add edges for both graphs
AddEdge graph1 n1 n2 e1
AddEdge graph2 n1 n2 e1

# Test children in both graphs
ListChildren graph1 n1
ListChildren graph2 n1