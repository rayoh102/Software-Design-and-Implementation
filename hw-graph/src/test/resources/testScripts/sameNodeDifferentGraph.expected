# Tests if we can create multiple instances of the same graph

# Create a graph
created graph graph1

# Create another graph
created graph graph2

# Add a node to first graph
added node n1 to graph1

# Add same node to second graph
added node n1 to graph2

# Repeat for another node
added node n2 to graph1
added node n2 to graph2

# Add edges for both graphs
added edge e1 from n1 to n2 in graph1
added edge e1 from n1 to n2 in graph2

# Test children in both graphs
the children of n1 in graph1 are: n2(e1)
the children of n1 in graph2 are: n2(e1)

